airbyte-webapp/src/scss/_theme-light.scss

```diff
@@ -91,9 +91,9 @@
   --color-overlay-background: rgba(26 25 77 / 50%);
 
   --box-shadow: 0 2px 4px rgba(26 25 77 / 12%);
-  --box-shadow-left: -2px 0 10px rgba(26 25 77, 12%);
-  --box-shadow-right: 2px 0 10px rgba(26 25 77, 12%);
+  --box-shadow-left: -2px 0 10px rgba(26 25 77 / 12%);
+  --box-shadow-right: 2px 0 10px rgba(26 25 77 / 12%);
```

Comment:
These box-shadow color values were just incorrect and made them not work. Fixed here to use a slash instead of a comma